newIDE/app/src/UI/ShareDialog/ShareLink.js

```diff
@@ -27,34 +27,28 @@ const ShareLink = ({ url }: Props) => {
   };
   return (
     <>
-      <TextFieldWithButtonLayout
-        noFloatingLabelText
-        renderTextField={() => (
-          <TextField
-            value={url}
-            readOnly
-            fullWidth
-            endAdornment={
-              <IconButton
-                onClick={onCopyLinkToClipboard}
-                tooltip={t`Copy`}
-                edge="end"
-              >
-                <Copy />
-              </IconButton>
-            }
-          />
-        )}
-        renderButton={style => (
-          <RaisedButton
-            primary
-            id="open-online-export-button"
-            label={<Trans>Open</Trans>}
-            onClick={onOpen}
-            style={style}
-          />
-        )}
-      />
+      <ResponsiveLineStackLayout alignItems="center" noMargin>
```

Comment:
TextFieldWithButtonLayout is mainly for instruction parameters, it has an automatic expand on the Line.
I prefered using this as we don't really need a text field (it's disabled)